tests/integration/cli/tests/run.rs

```diff
@@ -164,8 +164,10 @@ fn test_cross_compile_python_windows() -> anyhow::Result<()> {
             output.arg("-o");
             output.arg(python_wasmer_path.clone());
             output.arg(format!("--{c}"));
-            output.arg("--debug-dir");
-            output.arg(format!("{t}-{c}"));
```

Comment:
Do we still want this on CI?
If so, are you setting it via env var?

Reply:
I want it on the CI, just not locally.